bots/images/scripts/ovirt.setup

```diff
@@ -0,0 +1,151 @@
+OVESETUP_APACHE/configureRootRedirection=bool:True
+OVESETUP_APACHE/configureSsl=bool:True
+OVESETUP_VMCONSOLE_PROXY_CONFIG/vmconsoleProxyConfig=bool:True
+OVESETUP_CONFIG/websocketProxyConfig=bool:True
+OVESETUP_CONFIG/fenceKdumpListenerStopNeeded=bool:False
+OVESETUP_APACHE/enable=bool:False
+OVESETUP_CONFIG/engineServiceStopNeeded=bool:False
+OVESETUP_ASYNC/clearTasks=bool:False
+OVESETUP_DWH_CORE/enable=bool:False
+OVESETUP_DWH_PROVISIONING/postgresProvisioningEnabled=bool:False
+OVESETUP_CONFIG/imageioProxyConfig=bool:False
+# Don't install the the OVN provider:
+OVESETUP_OVN/ovirtProviderOvn=bool:False
+# Don't perform database full vacuum:
+OVESETUP_DB/engineVacuumFull=bool:False
+EOF
+
+/usr/bin/engine-setup --config=/answers.conf --offline --accept-defaults
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'false' where option_name = 'InstallVds';"
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'false' where option_name = 'EncryptHostCommunication';"
+runuser -u postgres -- psql engine -c "UPDATE vdc_options set option_value = 'true' where option_name = 'UseHostNameIdentifier';"
+/usr/bin/systemctl restart ovirt-engine
+
+# Setup a NFS server for SD
+mkdir -m777 /nfsexport
+echo "/nfsexport *(rw,sync)" > /etc/exports
+/usr/bin/systemctl enable --now nfs-server
+
+# Install VDSM
+rpm --import https://raw.githubusercontent.com/CentOS-Storage-SIG/centos-release-storage-common/master/RPM-GPG-KEY-CentOS-SIG-Storage
+yum install -y vdsm vdsm-client vdsm-hook-faqemu
+cat << 'EOF' > /etc/vdsm/vdsm.conf
+[vars]
+ssl = false
+fake_kvm_support = true
+EOF
+# WA until commit bf0f31e2b
+sed -i "s/pc-i440fx-rhel7.2.0/pc-i440fx-rhel7.3.0/g" /usr/libexec/vdsm/hooks/after_get_caps/10_faqemu
+sed -i "/def _graphics_spice_to_vnc(/i @workaround()" /usr/libexec/vdsm/hooks/before_vm_start/10_faqemu
+/usr/bin/vdsm-tool configure
+/usr/bin/systemctl start libvirtd
+/usr/bin/systemctl start vdsmd
+
+# Configure ovirtmgmt bridge
+echo '{"bondings": {}, "networks": {"ovirtmgmt": {"nic": "eth1", "netmask": "255.255.240.0", "ipaddr": "10.111.113.5", "gateway": "10.111.113.1", "defaultRoute": false}}, "options": {"connectivityCheck": false}}' | /usr/bin/vdsm-client --insecure -f - Host setupNetworks
+/usr/bin/vdsm-client --insecure Host setSafeNetworkConfig
+
+# add host, create storage on /nfsexport, attach it
+CURL="/usr/bin/curl -k -u admin@internal:engine -H Content-type:application/xml"
+DCID=$(${CURL} "https://${FQDN}/ovirt-engine/api/datacenters?search=name%3DDefault" | xmllint --xpath "string(/data_centers/data_center/@id)" -)
+${CURL} -d "<host> <name>host</name> <address>${FQDN}</address> <root_password>foobar</root_password> </host>" https://${FQDN}/ovirt-engine/api/hosts
+HOST_UP=""
+while [ -z ${HOST_UP} ]; do
+    sleep 5
+    HOST_UP=$(${CURL} "https://ovirt.cockpit.lan/ovirt-engine/api/hosts?search=status%3Dup" | xmllint --xpath "string(/hosts/host/@id)" -)
+done
```

Comment:
I'll remove this one. One of the customizations above actually bypass full host installation which can take ages since it yum installs packages on its own. Now it's just a round trip between two APIs, a fixed small delay should be fine